tests/integration/galley/analyze_test.go

```diff
@@ -34,6 +34,7 @@ const (
 	serviceRoleBindingFile = "testdata/servicerolebinding.yaml"
 	serviceRoleFile        = "testdata/servicerole.yaml"
 	invalidFile            = "testdata/invalid.yaml"
+	invalidExtensionFile   = "testdata/invalid.md"
```

Comment:
Could you add test cases for .yml and .json as well?

Reply:
Done. Added a test for json. Yaml is tested in existing testing cases.